gvsbuild/projects.py

```diff
@@ -405,6 +407,73 @@ def build(self):
         self.install(r'.\COPYING share\doc\glib-openssl')
         self.install(r'.\LICENSE_EXCEPTION share\doc\glib-openssl')
 
+@project_add
+class Project_gobject_introspection(GitRepo, Meson):
+    def __init__(self):
+        Project.__init__(self,
+            'gobject-introspection',
+            repo_url = 'https://git.gnome.org/browse/gobject-introspection',
+            fetch_submodules = False,
+            tag = 'wip/meson',
+            dependencies = [
+                'ninja',
+                'meson',
+                'msys2',
+                'pkg-config',
+                'glib',
+                # This ones are for add their's gir

```

Comment:
This is backwards? Those projects depend on `g-i` at build time.

Reply:
Yes, in the long run is probably better to have each project build the gir (if the g_ir_scanner is present or if we have a global configuration that say so). 
As more gtk project moves to meson probably the gir stuff is in there for free so it can be used easily (I need to make a couple of tests with json-glib & atk).

This one is for now to have something usable and to test all typelibs/gir.